src/api/lib/tasks/dev.rake

```diff
@@ -92,7 +94,24 @@ namespace :dev do
           sh 'rubocop -D -F -S --fail-level convention'
         end
       end
+
+      namespace :auto_gen_config do
+        task all: [:root, :rails] do
+        end
+        desc 'Autogenerate rubocop config in rails'
+        RuboCop::RakeTask.new(:rails) do |task|
+          task.options = ['--auto-gen-config', '--ignore_parent_exclusion']
+          task.fail_on_error = false
+        end
+        desc 'Autogenerate rubocop config in root'
+        task :root do
+          Dir.chdir('../..') do
+            sh 'rubocop --auto-gen-config || exit 0'
```

Comment:
Is it not possible to do this with `RuboCop::RakeTask`? :confused: 

why the `exit 0`?

Reply:
@Ana06 

> Is it not possible to do this with RuboCop::RakeTask? :confused:
>
> why the exit 0?

Please have a look at my [previous comment](#discussion_r197156750).

The `exit 0` is because rubocop returns `1` when it is run with the `--auto-gen-config` option.